stroeer/core/v1/article.proto

```diff
@@ -1079,3 +1090,34 @@ message BodyNode {
   */
   repeated Element elements = 5;
 }
+
+/**
+* @FileArticle Keyword
+*/
+
+/**
+* ```protobuf
+* message Keyword {}
+* ```
+* Extracted keywords from the article body like persons, locations, organizations etc.
+*/
+message Keyword {
```

Comment:
will this message be used outside of the article context? if not we could define it within the Article message.

Reply:
🤷  so far, it's only used in the article context